tests/unit/sagemaker/training_compiler/test_huggingface_pytorch_compiler.py

```diff
@@ -718,3 +718,31 @@ def test_register_hf_pytorch_model_auto_infer_framework(
     sagemaker_session.create_model_package_from_containers.assert_called_with(
         **expected_create_model_package_request
     )
+
+
+def test_accept_user_defined_environment_variables(
```

Comment:
I think we need to update these tests with the another version : 
FAILED tests/unit/sagemaker/training_compiler/test_huggingface_pytorch_compiler.py::test_accept_user_defined_environment_variables[4.21.1]
FAILED tests/unit/sagemaker/training_compiler/test_huggingface_pytorch_compiler.py::test_accept_user_defined_environment_variables[4.21] 


```
arg = '4.21.1'
available_options = ['4.6.1', '4.10.2', '4.11.0', '4.12.3', '4.17.0', '4.26.0', ...]
arg_name = 'huggingface version'

    def _validate_arg(arg, available_options, arg_name):
        """Checks if the arg is in the available options, and raises a ``ValueError`` if not."""
        if arg not in available_options:
>           raise ValueError(
                "Unsupported {arg_name}: {arg}. You may need to upgrade your SDK version "
                "(pip install -U sagemaker) for newer {arg_name}s. Supported {arg_name}(s): "
                "{options}.".format(arg_name=arg_name, arg=arg, options=", ".join(available_options))
            )
E           ValueError: Unsupported huggingface version: 4.21.1. You may need to upgrade your SDK version (pip install -U sagemaker) for newer huggingface versions. Supported huggingface version(s): 4.6.1, 4.10.2, 4.11.0, 4.12.3, 4.17.0, 4.26.0, 4.28.1, 4.6, 4.10, 4.11, 4.12, 4.17, 4.26, 4.28.
```

Reply:
I'm not sure I understand. Where is the version picked by the test defined?

I just rebased the pull request.